apps/speed.c

```diff
@@ -1073,25 +1072,26 @@ static int run_benchmark(int async_jobs, int (*loop_function)(void *), loopargs_
 
 
     for (i = 0; i < async_jobs && !error; i++) {
-        switch (ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
+        async = ASYNC_start_job(&(loopargs[i].inprogress_job), loopargs[i].wait_ctx,
```

Comment:
but it's an assignment, not in the middle of an expression.  i think it's more clear.